phpseclib/Crypt/RC4.php

```diff
@@ -99,14 +99,13 @@ class Crypt_RC4 extends Crypt_Base
     var $block_size = 0;
 
     /**
-     * The default password key_size used by setPassword()
+     * Key Length
```

Comment:
maybe mentioning this is the length in bytes might be useful for the description?